docs/can-guides/experiment/crud-beginner/crud-beginner.md

```diff
@@ -316,39 +319,42 @@ Let’s look at our code again:
 @sourceref ./7.js
 @highlight 21,only
 
-When the button emits a `click` event, the `save()` method on the ViewModel will be called.
+When the button emits a `click` event, the `save()` method on the custom element will be called.
 
-Again, you might be wondering where we’ve defined the `save()` method in the ViewModel… we’ll get there in just a moment. 😊
+Again, you might be wondering where we’ve defined the `save()` method in the custom element… we’ll get there in just a moment. 😊
 
 ### Defining custom properties
 
-Earlier we said that a:
+Earlier we said that:
 
-> [can-component.prototype.ViewModel `ViewModel`] is an object (with properties and methods) from which the _view_ gets its _model_ data.
+> [can-stache-define-element/static.define `static define`] returns an object that _defines_ the properties available to the view.
 
-This is true, although there’s more information to be known. A component’s ViewModel is actually an instance of
-[can-define/map/map DefineMap], which is an observable data type used throughout CanJS.
+This is true, although there’s more information to be known. The `static define` object is made up of [can-define-object DefineObject]-like
+property definitions that explicitly configure how a custom element’s properties are defined.
 
-We’ve been defining properties and methods on the ViewModel with the
+We’ve been defining properties and methods on the custom element with the
 [standard JavaScript getter and method syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions#Defining_method_functions).
-Now we’re going to use [can-define.types.propDefinition#String DefineMap’s string syntax] to define
-a property as a string:
+Now we’re going to use [can-define-object/object.types.property#function__ DefineObject’s constructor syntax] to define
+a property as a [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String):
 
 @sourceref ./7.js
-@highlight 32,only
+@highlight 35,only
 
-In the code above, we define a new `newName` property on the ViewModel. When this property is set,
+In the code above, we define a new `newName` property on the custom element. When this property is set,
```

Comment:
With `newName: String`, this will throw if the value is not a string. `newName: type.maybeConvert(String)` will do what is explained here. This might be a lot to explain though.

Reply:
I think I might change this paragraph to:

> In the code above, we define a new `newName` property on the custom element as a `String`. If this property is set to a value that’s not a String, CanJS will throw an error. If you instead want the value to be converted to a string, you could use [type.convert(String)](https://canjs.com/doc/can-type/convert.html).